azure-go-aks-multicluster/go.mod

```
@@ -0,0 +1,9 @@
+module azure-go-aks-multicluster
+
+go 1.15
+
+require (
+	github.com/pulumi/pulumi-azure-native/sdk v0.7.1
+	github.com/pulumi/pulumi-azuread/sdk/v3 v3.5.1
+	github.com/pulumi/pulumi/sdk/v2 v2.21.2
```

```suggestion
	github.com/pulumi/pulumi/sdk/v2 v2.24.1
```

Sadly Go will pin these versions and we are still looking for a good automated means to keep versions up-to-date.